docs/README.md

```diff
@@ -31,7 +31,7 @@ Optionally to enable Login to the dashboard
 ### Before you begin
 
 * You must enable basic auth to prevent user-functions from accessing the admin API of the gateway
-* A list of valid users is defined in the CUSTOMERS file in this GitHub repo, this acts as an ACL, but you can define your own
```

Comment:
Valid users are github or gitlab usernames? If you are installing this yourself, presumably you only want your own github username to begin with?